### DISCUSSION TITLE
Is there a way to make the background transparent?

### DISCUSSION THREAD

**Discussion by schererleander** [mason-org/mason.nvim#355] [Q&A] [answered]
Is there a way to make the background transparent?

<img width="960" alt="image" src="https://user-images.githubusercontent.com/80996591/187247598-507007d6-f07e-4a55-a052-09459dc7f9fc.png">

**Comment by williamboman** [comment, marked as answer]
Are you perhaps looking for `:h 'winblend'`?

```vim
autocmd FileType mason setlocal winblend=10
```

**Reply by rohit-kumar-j** [reply]
@williamboman, can It be done like the telescope plugin? The text from the buffer at the back is ovrlapping.

![image](https://user-images.githubusercontent.com/37873142/229679639-beae7af5-00cb-47bd-bfb1-96207d9c6cb2.png)

![image](https://user-images.githubusercontent.com/37873142/229679787-1062572f-8aa8-429a-9f49-e75a18331077.png)
